### ISSUE TITLE
0.8.x -- glitter - 401 for fonts in chrome

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#279]
per sab irc  channel:
```
[12:54am] <sayd_> getting 401s on the woff font files with Glitter on osx
[12:55am] <sayd_> and ttf
[01:48am] <sayd_> i think my server is listening on ipv6 at localhost:http-alt but *:http-alt for ipv4. since its a remote server that would possibly cause some issues.
[02:03am] <sayd_> its listening on both, but how would i disable ipv6
[02:04am] <sayd_> http://danceb.in/kDESnFJC5RGcQdaxDMIysQ/raw
[02:21am] <sayd_> something chrome specific is causing the fonts not to load
[02:27am] <sayd_> known issue https://code.google.com/p/chromium/issues/detail?id=516192
```

**Comment by thezoggy** [comment]
so i know you can fix this by using CORS headers (apache .htaccess) by Allowing cross-origin access to web fonts... 

so maybe something like:
`    cherrypy.response.headers["Access-Control-Allow-Origin"] = "*"`

**Comment by jeffgeorge** [comment]
As per the last link, this is a known bug currently present on the Chrome Dev Channel that only presents if your SABnzbd instance is behind http auth. Nothing we need to be worried about unless that issue goes awry and Google decides to consider this a feature.

**Comment by Safihre** [comment]
I also noticed Chrome is being very strict when using HTTPS without valid certificates, it will not load the font 10% of the time. Very randomly..

The CORS header will also solve my problem with the ports in the Config-restart (not allowing me to check the new port from the old port). But do we really want me to enable that? Seems like a something that should be off.

I will look into base64 encoding the files, instead of having external files!

**Comment by thezoggy** [comment]
@Safihre you just limit the scope of where you add that header.. so just /font for example

**Comment by Safihre** [comment]
Put them in the CSS as Base64, seems to work pretty well actually #280
This fixes any problems that Chrome has with fonts on auth/https connections that don't have valid certificates. Both Firefox and Chrome use Woff font's, so only for IE it will have to download 20kb extra of the CSS while for FF/Chrome it will lower the number of requests by 1.

**Comment by jeffgeorge** [comment]
I believe the underlying Chrome issue has been resolved in Chrome Build 45.0.2454.37, though it may no longer be relevant thanks to work in #280. Either way, this issue should be safe to close.
